src/EdgeOrder/examples/Get-AzEdgeOrder.md

```diff
@@ -0,0 +1,18 @@
+### Example 1: Get order details
+```powershell
+
+PS C:\> $order = Get-AzEdgeOrder -Name pwOrderItem11 -SubscriptionId "SubscriptionId" -Location "eastus" -ResourceGroupName "resourceGroupName"
+PS C:\> $order | fl
+
+CurrentStageDisplayName      :
+CurrentStageName             : Placed
+CurrentStageStartTime        : 11/16/2021 10:35:00 AM
+CurrentStageStatus           : Succeeded
+Id                           : /subscriptions/SubscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.EdgeOrder/locations/eastus/orders/pwOrderItem11
+Name                         : pwOrderItem11
+OrderItemId                  : {/subscriptions/SubscriptionId/resourceGroups/resourceGroupName/providers/Microsoft.EdgeOrder/orderItems/examplePowershell}
+OrderStageHistory            : {, }
+SystemData                   : Microsoft.Azure.PowerShell.Cmdlets.EdgeOrder.Models.Api20.SystemData
+Type                         : Microsoft.EdgeOrder/orders
+```
+
```

Comment:
Please make sure a short description is appended after each exception or when running ./build-modue.ps1 it will error out.
```suggestion
This example gets the details of an order by its resource group name, name and location.
```